homematic/data/run.sh

```diff
@@ -37,11 +35,11 @@ init_interface_list "$RF_ENABLE" "$HMIP_ENABLE" "$WIRED_ENABLE"
 # RF support
 if [ "$RF_ENABLE" == "true" ]; then
```

Comment:
Bashio can check this directly.

```suggestion
if bashio::config.true 'rf_enable'; then
```